app/screens/settings/about/about.tsx

```diff
@@ -171,6 +185,14 @@ const About = ({componentId, config, license}: AboutProps) => {
         popTopScreen(componentId);
     });
 
+    const copyToClipboard = () => {
```

Comment:
This should be inside a `useCallback` to avoid unneeded re-renders.
Probably it just needs to depend on the `intl`.

Reply:
Yes